src/protocols/secure_channel/tests/TestMessageCounterManager.cpp

```diff
@@ -152,13 +147,8 @@ nlTestSuite sSuite =
  */
 int Initialize(void * aContext)
 {
-    // Initialize System memory and resources
-    VerifyOrReturnError(chip::Platform::MemoryInit() == CHIP_NO_ERROR, FAILURE);
-    VerifyOrReturnError(gIOContext.Init(&sSuite) == CHIP_NO_ERROR, FAILURE);
-    VerifyOrReturnError(gTransportMgr.Init("LOOPBACK") == CHIP_NO_ERROR, FAILURE);
-
     auto * ctx = static_cast<TestContext *>(aContext);
-    VerifyOrReturnError(ctx->Init(&sSuite, &gTransportMgr, &gIOContext) == CHIP_NO_ERROR, FAILURE);
+    VerifyOrReturnError(ctx->Init(&sSuite) == CHIP_NO_ERROR, FAILURE);
```

Comment:
This file is going to need some DrainAndServiceIO, but at the moment it's not even being compiled/run, so it's not being an issue...  Just file a followup to track it, please?

Reply:
Left a comment in #14204